packages/wdio-browserstack-service/src/accessibility-handler.ts

```diff
@@ -179,28 +217,27 @@ class _AccessibilityHandler {
       * Cucumber Only
     */
     async beforeScenario (world: ITestCaseHookParameter) {
-        if (!this.shouldRunTestHooks(this._browser, this._accessibility)) {
-            return
-        }
-
         const pickleData = world.pickle
         const gherkinDocument = world.gherkinDocument
         const featureData = gherkinDocument.feature
         const uniqueId = getUniqueIdentifierForCucumber(world)
-        const shouldScanScenario = shouldScanTestForAccessibility(featureData?.name, pickleData.name, this._accessibilityOptions, world, true)
-        const isPageOpened = await this.checkIfPageOpened(this._browser, uniqueId, shouldScanScenario)
+        try {
+            if (!this.shouldRunTestHooks(this._browser, this._accessibility)) {
+                return
+            }
```

Comment:
only put the things that we expect to throw into try/catch

```suggestion
        if (!this.shouldRunTestHooks(this._browser, this._accessibility)) {
            return
        }
        try {
```